index.yaml

```diff
@@ -17430,6 +17430,136 @@ entries:
     - assets/rancher-logging-crd/rancher-logging-crd-3.6.000.tgz
     version: 3.6.000
   rancher-monitoring:
+  - annotations:
+      artifacthub.io/license: Apache-2.0
+      artifacthub.io/links: |
+        - name: Chart Source
+          url: https://github.com/prometheus-community/helm-charts
+        - name: Upstream Project
+           url: https://github.com/prometheus-operator/kube-prometheus
```

Comment:
There's an extra space before `url:`. Strictly speaking it's not a yaml error, as this block is a string, but it looks like it's intended to be input for some other yaml process, and therefore should be conforming yaml code